examples/migrate/cstor-volume-migration.yaml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  # the name can be of the form migrate-cstor-<pv-name> 
+  name: migrate-cstor-pvc-dd6ad226-c2b3-47c8-a8bf-46cb820d187a
+  namespace: openebs
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      #VERIFY the value of serviceAccountName is pointing to service account
+      # created within openebs namespace. Use the non-default account.
+      # by running `kubectl get sa -n <openebs-namespace>`
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name:  migrate
+        args:
+        - "cstor-volume"
+        # name of the pv that is to be migrated
+        - "--pv-name=pvc-dd6ad226-c2b3-47c8-a8bf-46cb820d187a"
+
+        #Following are optional parameters
+        #Log Level
+        - "--v=4"
+        #DO NOT CHANGE BELOW PARAMETERS
+        env:
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        tty: true
+        # the version of the image should be same the 
+        # version of cstor-operator installed.
+        image: openebs/migrate-amd64:1.12.0
+      restartPolicy: Never
```

Comment:
Q: Is there any specif reason for not having `OnFailure`?

Reply:
No specific reason just to get all the errors from different job pods